components/concept/storage/src/main/java/mozilla/components/concept/storage/Storage.kt

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.storage
+
+/**
+ * An interface which provides generic operations for storing browser data like history and bookmarks.
+ */
+interface Storage {
+
+    /**
+     * Internal database maintenance tasks. Ideally this should be called once a day.
```

Comment:
nit: remove last part of this comment; that applies to a specific places impl, not a generic interface.

Reply:
Done